tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Check that the service variable values are sane
+  assert:
```

Comment:
I added this assert to try to provide some user validation if they pass in strange combinations. I am sure this will not cover all possible combinations that could break the `win_service` task further on, but imo is a solid start.